### ISSUE TITLE
Unable to find font Calibri or fallback font DejaVu Sans!

### ISSUE THREAD

**Issue by amugox** [ClosedXML/ClosedXML#1902]
## Read and complete the full issue template

Do not randomly delete sections. They are here for a reason.

**Do you want to request a *feature* or report a *bug*?**
- [x] Bug
- [ ] Feature
- [ ] Question

**Did you test against the latest CI build?**
- [x] Yes
- [ ] No

If you answered `No`, please test with [the latest development build](https://ci.appveyor.com/project/ClosedXML/ClosedXML/branch/develop/artifacts) first.

**Version of ClosedXML**
0.97.0

**What is the current behavior?**

When I run on Linux environment I get this error:  Unable to find font Calibri or fallback font sans-serif. Install missing fonts or specify a different fallback font through 'LoadOptions.DefaultGraphicEngine = new DefaultGraphicEngine("Fallback font name")

**What is the expected behavior or new feature?**

Which font should I use or how do I go about this error?

**Comment by jahav** [comment]
tl;dr;: Pick an installed font/install a new font with width metrics closest to Calibri and follow the error message.

https://github.com/ClosedXML/ClosedXML/wiki/Graphic-Engine

The default font for a xlsx is a Calibri font. That font is copyrighted and isn't present on non-windows. The fallback font is a font that is used when a font is not found (not a calibri, any font that is not found on the particular machine) - on linux, there is no font ClosedXML can rely on (e.g. what fonts does AWS have installed for lambda functions - if any).
 
Find what fonts you have installed and pick a one that is close to Calibri. SixLabors.Fonts is looking into these folders on Linux ([source](https://github.com/SixLabors/Fonts/blob/main/src/SixLabors.Fonts/SystemFontCollection.cs#L38)): 
* %HOME%/.fonts/
* "/usr/local/share/fonts/"
* "/usr/share/fonts/"

Font is used only determine box size of a cell during `AdjustToContent`. At the version 0.97, the `AdjustToContent` is not particularly precise anyway. You options:

# Install Calibri
That is probably best, since that is the default font for worksheets. I have no idea about legality. Google "How to install Calibri on linux"

# Install a font
Find a font that looks similar to Calibri (and by similar I mean character widths, not shape of letters).

A quick google found (no endorsement, just a result):
> Carlito (Google’s Carlito font, google-crosextrafonts-carlito) is a modern, friendly sans-serif font, metric-compatible with Microsoft Fonts Calibri font. Carlito comes in regular, bold, italic, and bold italic. The family covers Latin-Greek-Cyrillic (not a complete set, though) with about 2,000 glyphs. It has the same character coverage as Microsoft Fonts Calibri. This font is sans-serif typeface family based on Lato. Carlito is a default Calibri font replace in the LibreOffice Suite.

We are interested in the **metric-compatible with Microsoft Fonts Calibri** part, so it might be a solid idea. It is not exact for numbers, but letters seem pretty close (upper Calibri, lower Carlito):
![image](https://user-images.githubusercontent.com/7634052/199342659-4bfba592-8b42-4739-89ad-d8abf3e69315.png)

# Use an existing font
Pick any font you already have installed and pick one that has looks similar to Calibri (probably slightly wider to avoid clipping or `####` signs).

**Comment by genifycom** [comment]
Please see #1947

**Comment by inkysquid** [comment]
If you don't care about the IXLGraphicEngine, then you can provide your own implementation and bypass the font wrangling.

```
var loadOptions = new LoadOptions
{
    GraphicEngine = NoGraphicsEngine.Instance
};
using var workbook = new XLWorkbook(stream, loadOptions);
```
...

```
internal class NoGraphicsEngine : IXLGraphicEngine
{
    private NoGraphicsEngine() { }
    public static NoGraphicsEngine Instance { get; } = new();
    public double GetDescent(IXLFontBase font, double dpiY) => default;
    public GlyphBox GetGlyphBox(ReadOnlySpan<int> graphemeCluster, IXLFontBase font, Dpi dpi) => default;
    public double GetMaxDigitWidth(IXLFontBase font, double dpiX) => default;
    public XLPictureInfo GetPictureInfo(Stream imageStream, XLPictureFormat expectedFormat) => default;
    public double GetTextHeight(IXLFontBase font, double dpiY) => default;
    public double GetTextWidth(string text, IXLFontBase font, double dpiX) => default;
}
```

**Comment by Ci5co** [comment]
Sorry to answer to an old post, but an alternative way is mentioned here: https://github.com/ClosedXML/ClosedXML/releases/tag/0.97.0. With this approach, you avoid changing each of your calls to the different overloads of `XLWorkbook` constructor 

Specifically within these sections
- Breaking changes 
  - **item** Non-windows environments
- Improvements
  - **sub-section** System.Drawing.Common removal

**Comment by inkysquid** [comment]
> Sorry to answer to an old post, but an alternative way is mentioned here:

Thanks @Ci5co Looking at the post, is this what you are referring to?

```
LoadOptions.DefaultGraphicsEngine = NoGraphicsEngine.Instance;
```
